package.json

```diff
@@ -26,27 +29,27 @@
   },
   "devDependencies": {
     "babel-cli": "^6.18.0",
-    "babel-plugin-transform-object-rest-spread": "^6.19.0",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-react": "^6.16.0",
-    "enzyme": "^2.6.0",
-    "happiness": "^7.1.2",
+    "babel-eslint": "^7.2.3",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "enzyme": "^2.9.1",
+    "happiness": "^10.0.2",
     "mocha": "^3.2.0",
     "mochify": "^2.18.1",
-    "react": "^15.6.0",
-    "react-addons-test-utils": "^15.6.0",
-    "react-dom": "^15.6.0"
+    "react-test-renderer": "^16.1.1"
```

Comment:
I don't think we need this anymore, I'll check. There was some funny business with Enzyme that I installed some dependencies for and ended up not needing, this may have been one.

Reply:
Okay, I'll change those. Is this something we want to begin doing for all of our peer dependencies?